Automattic/pocket-casts-android
# Playback 2023 - Longest episode story

| 📘 Part of: #1463 | 🎨 Designs: lH66LwxxgG8btQ8NrM0ldx-fi-1599_20385 |
|:---:|:---:|

## Description
Adds the 2023 visual for the Longest Episode Story.

Animations are not part of this task and will be tackled later.

## Testing Instructions
1. Make sure you're logged in to an account that has a few episodes listened
3. Go to Profile
4. Tap the EOY image
5. Skip to the longest episode story
5. ✅ Check that the story looks good, that it shows the longest episode you listened to
6. Share this story
8. ✅ The story image asset should look good

## Screenshots or Screencast 

Pixel 7| Pixel 7 (long titles) 
|----|-----|
<img width = 300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/8846d87c-3c38-4f6e-95dc-b68bc72b532e"/> | <img width = 300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/26fa0fca-d370-42d0-a8b1-9713c271c08b"/>

Tablet (Pixel 9)
|----|
|<img width = 450 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/19373ef7-f48d-4ba4-8b12-481d492a82c1"/>?


## Checklist
 
#### I have tested any UI changes...
